llvm/lib/Target/Xtensa/XtensaFrameLowering.cpp

```diff
@@ -255,19 +256,42 @@ void XtensaFrameLowering::determineCalleeSaves(MachineFunction &MF,
     SavedRegs.set(FP);
 }
 
+static unsigned estimateFunctionSizeInBytes(const MachineFunction &MF,
+                                            const XtensaInstrInfo &TII) {
+  unsigned FnSize = 0;
+  for (auto &MBB : MF) {
+    for (auto &MI : MBB) {
```

Comment:
Account for block (and function) alignment?

Reply:
Thank you very much for comment. I've implemented function size calculation  like it is done in BranchRelaxation. Also I corrected insertIndirectBranch function.